utils/build.py

```diff
@@ -49,18 +49,13 @@ def run(cmd, cwd, env, justprint):
         raise RuntimeError('Failed to run %s in %s' % (cmd, cwd))
 
 
-def quote(string):
-    return '"%s"' % string.replace('"', '\\"')
-
-
-def quote_some(command, indices):
-    """Transforms command (a list of strings) into a single string by
-    joining its elements, but with quotes around those elements in the
-    indices list.
+def parse_output_into_env(command, cwd):
+    """Runs command in cwd, expecting it to produce name=value lines on
+    output.  Parses those lines into a dictionary object.
     """
-    quoted = [(quote(e) if i in indices else e)
-              for (i, e) in enumerate(command)]
-    return ' '.join(quoted)
+    p = subprocess.Popen(command, stdout=subprocess.PIPE, cwd=cwd, shell=True)
+    return dict([tuple(line.split('='))
```

Comment:
split is wrong here.  
For example, 
  'FOO=BAR=BAZ'.split('=)
will make a list of 3 elements.  You should only split on the first equal sign.